src/audio/dai.c

```diff
@@ -615,7 +615,8 @@ static int dai_copy(struct comp_dev *dev)
 			    dev->params.direction);
 		platform_dai_wallclock(dev, &dd->wallclock);
 
-		return ret;
+		/* let's not copy further */
+		return 1;
```

Comment:
Just wondering (for a future PR), do we want to define a macro for PIPELINE_COPY_FURTHER_STOP to make it obvious for anyone reading the 1 here.

Reply:
Yes, I was thinking about that and it would also be helpful for things like prepare, so probably I will make a PR with something like PIPELINE_OPERATION_FURTHER_STOP.